Solutions/Feedly/Data Connectors/FeedlySentinelConnector/worker.py

```diff
@@ -1,12 +1,11 @@
 import logging
 import time
-from collections.abc import Iterable
+from typing import List, Dict, Any
 
 from FeedlySentinelConnector.config import FeedlySentinelConfig
 from FeedlySentinelConnector.feedly_downloader import FeedlyDownloader
 from FeedlySentinelConnector.sentinel_api import SentinelApiConnector
 from FeedlySentinelConnector.state_manager import FeedlyCheckpoint
-from pandas import DataFrame, concat
```

Comment:
pandas is overkill given use case, and connector should be more lightweight / faster to install